src/demo-app/table/custom-table/custom-table.scss

```diff
@@ -1,8 +1,9 @@
-.mat-table {
-  height: 300px;
-  overflow: auto;
+.mat-table, wrapper-table {
+  width: 100%;
 }
 
 .mat-card {
   margin: 8px 0;
+  height: 300px;
```

Comment:
alphabetical order.

Reply:
We don't have a rule for css property order, actually